client-v2/src/components/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const AppContainer = styled('div')`
```

Comment:
you can also use <code>styled.div``</code>

Reply:
I would like to but there’s currently an issue with Flow when using that method 😞 https://github.com/styled-components/styled-components/issues/1321 - another reason to use TS!